ftw/referencewidget/profiles/default_plone5/registry.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<registry>
+    <records interface="ftw.referencewidget.interfaces.IReferenceSettings" />
+
+    <records prefix="plone.resources/referencewidget_css" interface="Products.CMFPlone.interfaces.IResourceRegistry">
+        <value key="css">
+            <element>++resource++ftw.referencewidget/refwidget.css</element>
```

Comment:
This can be merged into the `refwidget_js` resource. I'd then rename it to `ftw.referencewidget`.
The docu says:

> A resource consists of one JavaScript and either none, one or multiple Less or CSS files.

https://docs.plone.org/adapt-and-extend/theming/resourceregistry.html#resources-and-bundles